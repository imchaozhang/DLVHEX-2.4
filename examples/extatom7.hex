% simple auxiliary input rule test

input(a).
input(b).

intermediate(B) :- &testConcat[A,x](B), input(A).

output(C) :- &testConcat[B,y](C), intermediate(B).
