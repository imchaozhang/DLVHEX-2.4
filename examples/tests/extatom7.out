{intermediate(bx), intermediate(ax), output(axy), output(bxy)}
